src/miner.erl

```diff
@@ -413,8 +414,23 @@ handle_info({blockchain_event, {add_block, Hash, Sync, _Ledger}},
                                   ConsensusGroup, {next_round, NextRound,
                                                    blockchain_block:transactions(Block),
                                                    Sync}),
-                                set_next_block_timer(State#state{current_height = Height});
-
+                                {ok, ConsensusAddrs} = blockchain_ledger_v1:consensus_members(Ledger),
+                                BBAs = blockchain_utils:bitvector_to_map(length(ConsensusAddrs), blockchain_block_v1:bba_completion(Block)),
+                                BBAAddrs = maps:fold(fun(K, true, Acc) ->
```

Comment:
I think that we should check the size of `BBAAddrs` and if it's less than 2f+1 we should not actually go early.  sometimes there are still 0 bba votes, we need to track those down, but when they happen they would cause the whole group to go immediately, causing a short block.